CMakeLists.txt

```diff
@@ -39,6 +39,8 @@ option(RDK_USE_BOOST_SERIALIZATION "Use the boost serialization library if avail
 option(RDK_BUILD_TEST_GZIP "Build the gzip'd stream test" OFF)
 option(RDK_OPTIMIZE_NATIVE "Use native features while compiling." ON)
 option(RDK_USE_STRICT_ROTOR_DEFINITION "Use the most strict rotatable bond definition" ON)
+option(RDK_BUILD_FREESASA_SUPPORT "build the rdkit freesasa wrapper" ON )
```

Comment:
I don't think we should default to ON until we have a bit more experience with the package.